DigitalRain/Raindrops/ColorCalculator.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+
+namespace DigitalRain.Raindrops
+{
+    public class ColorCalculator
+    {
+        private static int _maxAlpha = 255;
+
+        private readonly double _timespan;
+        private readonly Color _lerpStartColor;
+        private readonly Color _lerpEndColor;
+        private readonly float _lerpTime;
+        public Color StartColor { get { return _lerpStartColor; } }
+
+        public ColorCalculator(double timespan, Color startColor, Color endColor, float lerpTime)
```

Comment:
I love it. I was actually thinking about abstracting all this color stuff after our conversation last night. +1